frontend/components/atoms/AccountSettings/BoxedListItem.js

```diff
@@ -0,0 +1,19 @@
+import {SystemComponent} from '../SystemComponents';
+
+
+const BoxedListItem = ({className, variant, text}) => {
```

Comment:
You could make this:
```javascript
const BoxedListItem = styled(SystemComponent).attrs({
  pt: 1,
  ...
})
```
(I think this will work)

Reply:
We probably want to keep bg prop as `bg` instead of `variant` since `variant` isn't clear on what it is